src/core/public/chrome/chrome_service.tsx

```diff
@@ -12,6 +12,7 @@ import { BehaviorSubject, combineLatest, merge, Observable, of, ReplaySubject }
 import { flatMap, map, takeUntil } from 'rxjs/operators';
 import { parse } from 'url';
 import { EuiLink } from '@elastic/eui';
+import { valid } from 'semver';
```

Comment:
I suspect this and the `from '..'` below could be the cause of the major jump. I think it makes sense to accept the package info version directly below and then just assume that it's a valid semver version rather than importing `semver` to validate it client-side.